### ISSUE TITLE
Non-English doesn't work

### ISSUE THREAD

**Issue by YaLTeR** [WuTheFWasThat/vimflowy#351]
I tried switching to the Russian keyboard layout but that just continued typing in English letters.

**Comment by WuTheFWasThat** [comment]
yeah this is a known issue.  pull requests welcome!  someone had attempted, see https://github.com/WuTheFWasThat/vimflowy/pull/320

**Comment by BettyJJ** [comment]
Also all languages that need an IME (input method) do not work, e.g. Chinese, Japanese, Korean, etc. 
I just found this project and loved it. Then I found that I couldn't input anything in my language, which is a dealbreaker. 😢

**Comment by WuTheFWasThat** [comment]
sorry :( I don't have time to figure it out but would love if someone could contribute it (#320 could be a good starting point)

**Comment by BettyJJ** [comment]
Sorry, I just tried again and found that IME actually works. It just works weirdly. Its status bar is either not displaying at its usual position or not displaying at all. So it's not impossible, just very difficult to input these languages.

**Comment by veryuniquenickname** [comment]
Hi there,

I would like to follow up on this one, it would be really awesome to have an ability to just type in non-English while in INSERT mode. It's 2022, ASCII is a bit outdated :)
